### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by joe-bowman**
Closes https://github.com/ChorusOne/hipparchus/issues/27 and https://github.com/ChorusOne/hipparchus/issues/28

### CODE REVIEW

File: x/auth/keeper.go [hunk 1]
```diff
@@ -18,7 +18,8 @@ import (
 // encoding/decoding library.
 type AccountKeeper struct {
 	// The (unexposed) key used to access the store from the Context.
-	key sdk.StoreKey
+	key       sdk.StoreKey
+	supplyKey sdk.StoreKey
```
Comment (ParthDesai) [inline x/auth/keeper.go]: Why was this added?

**Comment by joe-bowman** [comment]
That was unintended. Feel free to remove

File: x/staking/keeper/slash.go [hunk 2]
```diff
@@ -217,6 +217,7 @@ func (k Keeper) slashUnbondingDelegation(ctx sdk.Context, unbondingDelegation ty
 		entry.Balance = entry.Balance.Sub(unbondingSlashAmount)
 		unbondingDelegation.Entries[i] = entry
 		k.SetUnbondingDelegation(ctx, unbondingDelegation)
+		k.ExportUnbondingsForAccount(ctx, unbondingDelegation.DelegatorAddress)
```
Comment (ParthDesai) [inline x/staking/keeper/slash.go:220]: We are already exporting at end of function `SetUnbondingDelegation`

**Comment by joe-bowman** [comment]
Well spotted ;)

File: x/staking/keeper/delegation.go [hunk 3]
```diff
@@ -713,6 +726,8 @@ func (k Keeper) CompleteUnbonding(ctx sdk.Context, delAddr sdk.AccAddress,
 		k.SetUnbondingDelegation(ctx, ubd)
 	}
 
+	k.ExportUnbondingsForAccount(ctx, ubd.DelegatorAddress)
```
Comment (ParthDesai) [inline x/staking/keeper/delegation.go:730]: We are already exporting at end of function `SetUnbondingDelegation`

Better to add exporting in `RemoveUnbondingDelegation`

**Review by joe-bowman — COMMENTED** [review]
lgtm.
